Mathlib/Topology/Constructions.lean

```diff
@@ -1818,6 +1818,13 @@ theorem continuous_uLift_down [TopologicalSpace X] : Continuous (ULift.down : UL
 theorem continuous_uLift_up [TopologicalSpace X] : Continuous (ULift.up : X → ULift.{v, u} X) :=
   continuous_induced_rng.2 continuous_id
 
+@[continuity]
+theorem continuous_uLift_map [TopologicalSpace X] [TopologicalSpace Y]
```

Comment:
```suggestion
theorem continuous_uliftMap [TopologicalSpace X] [TopologicalSpace Y]
```
apparently? And similarly `continuous_uliftUp` for the previous theorem (which I know you didn't write!)